ratom/views.py

```diff
@@ -83,6 +83,7 @@ def account_detail(request, pk):
 
 
 @api_view(["GET"])
+@permission_classes([IsAuthenticated])
 def message_detail(request, pk):
     """
     Retrieve a Message
```

Comment:
Shouldn't this be in api/views?

Reply:
api/views doesn't exist yet on ``develop``